ball_collision/scripts/utils.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Generates random number between min and max (inclusive) by exlcuding 0.
+ *
+ * @param {number} min Minimum number
+ * @param {number} max Maximum number
+ * @returns {number} A random number
+ */
+function getRandomNumber(min = 0, max = 0) {
+  let num = 0;
+  while (num === 0) {
+    num = Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+  return num;
+}
+
+/**
+ * Generates a random hex color code.
+ *
+ * @returns {string} Hex color code
+ */
+function getRandomColor() {
+  const colorCode = Math.floor(Math.random() * 16777215).toString(16);
```

Comment:
Me as a reviewer might not know what 16777215 stands for. Could you add this in a variable with name (like totalColorCombination) that defines what this value means?